### PR TITLE
add NVB logo and info

### PR DESCRIPTION
**Pull request by lcpliu**
https://bscscan.com/tx/0x4380cb9101500a694d58dd30767f30eb3b0bc107f60f2f313291074b87ffef03
https://bscscan.com/tx/0xcbb23b4cad147180ce8361ea769dea50d55ce85488674abeed8e22cb45b0fb2e

### CODE REVIEW

File: bsc/0x033e6fF2201FFE4Cf5550a4146BA0Bbc0677dC1c/logo.png
Comment (JulioCesarXY) [inline bsc/0x033e6fF2201FFE4Cf5550a4146BA0Bbc0677dC1c/logo.png:1]: Wrong Contract❌

File: 0xbAca410853B55FB4E6966Ac8c17acC840062DF2D/info.json
Comment (JulioCesarXY) [inline 0xbAca410853B55FB4E6966Ac8c17acC840062DF2D/info.json:1]: Delete this Wrong Directory 👍🏽

File: 0xbAca410853B55FB4E6966Ac8c17acC840062DF2D/logo.png
Comment (JulioCesarXY) [inline 0xbAca410853B55FB4E6966Ac8c17acC840062DF2D/logo.png:1]: Delete This Wrong Directory 👍🏽

**Review by JulioCesarXY — APPROVED** [review]

**Comment by lcpliu** [comment]
NVB This is the project I am submitting

**Comment by lcpliu** [comment]
When will the NVB I submitted be completed? It has been many days. Please complete it. Thank you.

**Comment by lcpliu** [comment]
Hello, I have submitted it for almost two weeks and it has not been approved yet

**Comment by lcpliu** [comment]
hi，When will the submitted project be completed?

**Pull request closed without merging**